repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/RTScanner.java

```diff
@@ -223,6 +229,22 @@ private void addInTempBlackList(Repository repository, String comment) {
         this.tempBlackList.put(repository.getId(), expirationDate);
     }
 
+    /**
+     * Give to build to inspectBuilds and it will submit to 
+     * ActiveMQ queue in KUBERNETES Mode if it's interesting.
+     */
+    public void submitIfBuildIsInteresting(Build build) {
+        this.initPipelineMode("tcp://localhost:61616","TestQueue");
```

Comment:
we should use the configured value in RepairnatorConfig

Reply:
That line actually can be completely removed :) . Thank you very much for the nice review as always.